MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -713,6 +719,12 @@ public bool CanRefreshPrimary(Type type)
                             }
                         }
 
+                        // only delete existing multi-images if new ones were added
+                        if (oldBackdropImages.Length > 0 && oldBackdropImages.Length < item.GetImages(ImageType.Backdrop).Count())
```

Comment:
Wouldn't this be a problem if you adobt your referenced backdrops to an amount lower than the current one? In that case the changed backdrops would be dropped instead of applied.
Ideally we'd have a flag on images for deciding if they originate from a provider or were added by a user, so we can remove all from the providers but keep the manually added user images.